### PR TITLE
Ruby Capstone Project: Catalog of things

### PR DESCRIPTION
**Pull request by emhamza**
### This is our capstone project for the Ruby module. In this milestone my team members and I have managed to accomplished the following tasks in the Group as well as individual activities 😃 

- [x] Set up Kanban Board for the project
- [x] Config Linters in order to follow the standards
- [x] Create a data base following the UML diagram
- [x] Create multiple classes, methods and their relations as per the mentioned requirement
- [x] Create a User Interface
- [x] Create unit test using spec for every individual file
- [x] Add a feature which preserve data in a `data` folder in the of JSON
- [x] Create schema sql to show the table classifications

**Here you can see the video presentation of or project** 💯 
_[CLICK THIS LINK](https://drive.google.com/file/d/12lw8GVFDXuJshdAAAhn7SGM6l966XmOL/view)_

- If you like the project please feel free to give a star to this repo 😀

### CODE REVIEW

File: schema/schema.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  cover_state VARCHAR(255),
+  FOREIGN KEY (id) REFERENCES label(id)
+);
+
+-- schema for labels
+CREATE TABLE Label (
+  id INT PRIMARY KEY,
+  title VARCHAR(255),
+  color VARCHAR(255)
+);
```
Comment (ShadyShawkat) [inline schema/schema.sql:73]: - [x] Please implement the `item` class as shown in the given UML diagram.
- [x] Kindly make sure to create indexes for each of the created foreign keys.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema/schema.sql (hunk 1 above)
Comment (emhamza) [reply schema/schema.sql:73]: @ShadyShawkat Sure thing! 👍  Let us see what did we do wrong 🔸

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**